beetsplug/fetchart.py

```diff
@@ -425,8 +488,11 @@ def __init__(self):
             'sources': ['coverart', 'itunes', 'amazon', 'albumart'],
             'google_key': None,
             'google_engine': u'001442825323518660753:hrh5ch1gjzm',
+            'fanarttv_project_key': None,
```

Comment:
should be a global variable as it's not expected to be overwritten in the config file